xbmc/cores/VideoPlayer/DVDCodecs/Audio/DVDAudioCodecPassthrough.cpp

```diff
@@ -134,7 +133,7 @@ bool CDVDAudioCodecPassthrough::AddData(const DemuxPacket &packet)
       return true;
 
     m_dataSize = m_bufferSize;
-    used = m_parser.AddData(pData, iSize, &m_buffer, &m_dataSize);
+    int used = m_parser.AddData(pData, iSize, &m_buffer, &m_dataSize);
```

Comment:
```suggestion
    auto used = m_parser.AddData(pData, iSize, &m_buffer, &m_dataSize);
```

Reply:
there was a discussion around this sort of change not too long ago. I think i am on the side of ints a basic type, why go auto when its longer than int to type